src/backend/metal/src/native.rs

```diff
@@ -17,7 +17,7 @@ use spirv_cross::msl;
 /// depend on pipeline layout, in which case the value would become `Compiled`.
 #[derive(Debug)]
 pub enum ShaderModule {
-    Compiled(metal::Library),
+    Compiled { library: metal::Library, remapped_entry_point_names: HashMap<String, String> },
```

Comment:
nit: format in multiple lines, like a struct

Reply:
Fixed. Is there a plan to run `rustfmt` eventually? I tried to run it on this file but there were too many diffs 😃